src/ProcessService/ProcessServiceUtils.cpp

```diff
@@ -191,9 +191,17 @@ std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() {
 
   std::vector<std::string_view> splits = absl::StrSplit(first_line, ' ', absl::SkipWhitespace{});
 
+  auto begin_field = splits.begin() + 1;  // The first field is the cpu ID `cpuXX`, so we skip that.
+  auto end_field = std::min(begin_field + 8, splits.end());
+  // We accumulate up to 8 fields. (Depending on what kernel version is available there might be
```

Comment:
(Super irrelevant: I would expect this comment to be before `end_field`...?)

Reply:
Yes. Thx. Done.